infra/MAINTAINERS.csv

```diff
@@ -0,0 +1,7 @@
+Name,Email,Github Username
+Adam Korcz,<adam@adalogics.com>,AdamKorcz
+David Korczynski,<david@adalogics.com>,DavidKorczynski
+Dongge Liu,<donggeliu@google.com>,Alan32Liu
+Holly Gong,<gongh@google.com>,hogo6002
+Jonathan Metzman,<metzman@google.com>,jonathanmetzman
+Oliver Chang,<ochang@google.com>,oliverchang
```

Comment:
Last comment: is there any benefit to keeping the "<>" for the email now that we're just using CSV?